gammapy/spectrum/models.py

```diff
@@ -1581,3 +1581,105 @@ def evaluate(self, energy, **kwargs):
 
         unit = 1 / (energy.unit * u.cm ** 2 * u.s)
         return dnde.to(unit)
+
+
+class Gaussian(SpectralModel):
+    r"""Gaussian spectral model.
+
+    .. math::
+
+        \phi(E) = \frac{N_0}{\sigma \sqrt{2\pi}}  \exp{ \frac{\left( E-\bar{E} \right)^2 }{2 \sigma^2} }
+
+    This is consistent
+    with the `Fermi Science Tools
+    <https://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/source_models.html#Gaussian>`_
+    and `ctools
+    <http://cta.irap.omp.eu/ctools/users/user_manual/getting_started/models.html#gaussian>`_.
+
+
+    Parameters
+    ----------
+    norm : `~astropy.units.Quantity`
+        :math:`N_0`
+    mean : `~astropy.units.Quantity`
+        :math:`\bar{E}`
+    sigma : `~astropy.units.Quantity`
+        :math:`\sigma`
+
+
+    Examples
+    --------
+    This is how to plot the default `Gaussian` spectral model:
+
+    .. code:: python
+
+        from astropy import units as u
+        from gammapy.spectrum.models import Gaussian
+
+        gaussian = Gaussian()
+        gaussian.plot(energy_range=[0.1, 100] * u.TeV)
+        plt.show()
+    """
+
+    def __init__(
+            self, norm=1e-12 * u.Unit("cm-2 s-1"), mean=1 * u.TeV, sigma=2 * u.TeV
+    ):
+        self.norm = Parameter("norm", norm)
+        self.mean = Parameter("mean", mean)
+        self.sigma = Parameter("sigma", sigma)
+
+        super().__init__([self.norm, self.mean, self.sigma])
+
+    @staticmethod
+    def evaluate(energy, norm, mean, sigma):
+        return (
+                norm
+                / (sigma * np.sqrt(2 * np.pi))
+                * np.exp((energy - mean) ** 2 / (2 * sigma ** 2))
+        )
+
+    def integral(self, emin, emax, **kwargs):
+        r"""Integrate Gaussian analytically.
+
+        .. math::
+            F(E_{min}, E_{max}) = \frac{N_0}{2} \left( erf(\frac{E_{max}-
+            \bar{E}}{\sqrt{2} \sigma}) - erf(\frac{E_{min}-\bar{E}}{\sqrt{2} \sigma}) \right)
+
+
+        Parameters
+        ----------
+        emin, emax : `~astropy.units.Quantity`
+            Lower and upper bound of integration range
+        """
+        from scipy.special import erf
+
+        # kwargs are passed to this function but not used
+        # this is to get a consistent API with SpectralModel.integral()
+        pars = self.parameters
+        u_min = (emin - pars["mean"].quantity) / (np.sqrt(2) * pars["sigma"].quantity)
+        u_max = (emax - pars["mean"].quantity) / (np.sqrt(2) * pars["sigma"].quantity)
+
+        return pars["norm"].quantity / (2) * (erf(u_max) - erf(u_min))
+
+    def energy_flux(self, emin, emax):
+        r"""Compute energy flux in given energy range analytically.
+
+        .. math::
```

Comment:
For clarity you might give the integral in the form of 
[ primitive function ]_Emin^Emax